packages/router/src/page-loader.tsx

```diff
@@ -101,6 +101,8 @@ export class PageLoader extends React.Component<Props> {
       slowModuleImport: false,
       params: props.params,
     })
+
+    window.scrollTo(0, 0)
```

Comment:
Careful, careful :) Prerender will not like this change!

```suggestion
    typeof window !== 'undefined' && window.scrollTo(0, 0)
```

Reply:
@dac09 You've got your work cut out for you now, don't you, scanning for stuff like this in every PR we do 😅

Reply:
Not looking forward to it! But anything `router` in particular, we have to watch out for!

Reply:
Here's another way actually (please do verify it works)
```suggestion
    global?.scrollTo(0, 0)
```

Reply:
this line, `global?.scrollTo(0, 0)`, works @dac09